plugins/GSdx/GS.cpp

```diff
@@ -475,6 +475,11 @@ static int _GSopen(void** dsp, const char* title, GSRendererType renderer, int t
 	return 0;
 }
 
+EXPORT_C_(void) GSosdLog(const char *utf8)
+{
+	if(s_gs) if(s_gs->m_dev) s_gs->m_dev->m_osd.Log(utf8);
+}
```

Comment:
yes thats what i meant, i've been distracted while doing this and have to keep taking breaks and coming back.  I'll fix it in the next push.